controller/destination/listener.go

```diff
@@ -170,13 +171,21 @@ func (l *endpointListener) toAddrSet(addresses []*updateAddress) *pb.AddrSet {
 	return &pb.AddrSet{Addrs: addrs}
 }
 
-func (l *endpointListener) getAddrMetadata(pod *coreV1.Pod) (map[string]string, *pb.TlsIdentity) {
+func (l *endpointListener) getAddrMetadata(pod *coreV1.Pod) (map[string]string, *pb.ProtocolHint, *pb.TlsIdentity) {
 	controllerNs := pkgK8s.GetControllerNs(pod)
 	ownerKind, ownerName := l.ownerKindAndName(pod)
 	labels := pkgK8s.GetPodLabels(ownerKind, ownerName, pod)
 
+	hint := &pb.ProtocolHint {}
```

Comment:
If you run [gofmt](https://golang.org/cmd/gofmt/) on this file it will fix the spacing issue here (no space between the struct name and initialization bracket).